canonical-web-and-design/ubuntu.com
# Update design of server and desktop download pages

## Done

- Add suru header and vanilla's accordian pattern to the server and desktop download thank-you pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/download/desktop/thank-you?version=18.04.2&architecture=amd64 and http://0.0.0.0:8001/download/server/thank-you?version=18.04.2&architecture=amd64
- **Cancel the download**
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that it matches the [design](#1308) except that it uses vanilla 1.8's accordian


## Issue / Card

Fixes [1308](https://github.com/canonical-web-and-design/web-squad/issues/1308)

## Screenshots

### closed

![image](https://user-images.githubusercontent.com/441217/62135831-d2289380-b2da-11e9-8efe-83917e9d0242.png)


### open

![image](https://user-images.githubusercontent.com/441217/62135856-deacec00-b2da-11e9-8170-c6abdb1cd4fe.png)
